types/sparkly/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for sparkly 3.1
+// Project: https://github.com/sindresorhus/sparkly
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = sparkly;
+
+declare function sparkly(numbers: any[], options?: sparkly.Options): string;
```

Comment:
The docs say 
```
sparkly(numbers, [options])
numbers
Type: array of number

Numbers to create the sparkline from.
```
I'm pretty sure numbers should be `number[]`. (Even if it doesn't throw, per sey, on other input, it's not intended to be useful with other inputs)